src/flow/src/batching_mode/frontend_client.rs

```diff
@@ -223,12 +228,32 @@ impl FrontendClient {
                     peer: db.peer.clone(),
                 });
 
-                db.database
-                    .handle(req.clone())
-                    .await
-                    .with_context(|_| InvalidRequestSnafu {
-                        context: format!("Failed to handle request: {:?}", req),
-                    })
+                let mut retry = 0;
+
+                loop {
+                    let ret = db.database.handle(req.clone()).await.with_context(|_| {
+                        InvalidRequestSnafu {
+                            context: format!("Failed to handle request: {:?}", req),
+                        }
+                    });
+                    if let Err(err) = ret {
```

Comment:
nitpick: Do we need to check the status code of the error `StatusCode::is_retryable()` to ensure we can retry this? e.g. It's not invalid argument error.

Reply:
#5987